packages/ihe-gateway-sdk/src/models/shared.ts

```diff
@@ -86,6 +86,8 @@ export const baseResponseSchema = z.object({
   externalGatewayPatient: externalGatewayPatientSchema.optional(),
   patientId: z.string().nullish(),
   operationOutcome: operationOutcomeSchema.optional(),
+  retried: z.number().optional(),
+  iheGatewayV2: z.boolean().optional(),
```

Comment:
Where is this used? Just a flag on the payload so we know when looking at logs? Maybe add a comment on these that are not obvious from usage?

Reply:
A flag on the payload so
a. when you look at results in the db you know if they are from v1 or v2
b. to run future analysis to see how much retrying logic is helping